client/src/views/ProjectsView/ProjectsView.tsx

```diff
@@ -108,6 +187,9 @@ const ProjectsView = (): ReactElement => {
 
   return (
     <>
+      <ViewTitle className={styles.viewTitle}>
+        {t('viewTitle', { epochNumber: currentEpoch })}
```

Comment:
When AW is open or AW is closed and `areCurrentEpochsProjectsHiddenOutsideAllocationWindow` is true we need to show `currentEpoch -1`.